pkg/controller/sparkapplication/controller.go

```diff
@@ -52,13 +52,16 @@ import (
 	crdlisters "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/listers/sparkoperator.k8s.io/v1alpha1"
 	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/config"
 	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/util"
+	"golang.org/x/time/rate"
 )
 
 const (
 	sparkDriverRole           = "driver"
 	sparkExecutorRole         = "executor"
 	sparkExecutorIDLabel      = "spark-exec-id"
 	podAlreadyExistsErrorCode = "code=409"
+	queueTokenRefillRate      = 25
```

Comment:
I think this is a little low, we might want to eventually add a queue for the pods that de-dupes updates for the same sparkApp. I think adjust this is a good idea